build.sbt

```diff
@@ -39,7 +39,7 @@ libraryDependencies ++= Seq(
 )
 
 // Wartremover
-wartremoverErrors ++= Warts.unsafe
+wartremoverErrors ++= Warts.unsafe.filterNot(_.clazz == Wart.Throw.clazz)
```

Comment:
We can, though `Warts.unsafe` are described as the "stable" checks (i.e. less false-positives). `throw` is triggered by pattern matching.